src/main/java/com/exasol/adapter/document/documentfetcher/files/TooManyRequestsException.java

```diff
@@ -0,0 +1,8 @@
+package com.exasol.adapter.document.documentfetcher.files;
+
+/**
+ * Exception thrown by the iterator returned by {@link FileLoader} if the source throttled the request.
+ */
+public class TooManyRequestsException extends RuntimeException {
```

Comment:
This is only thrown from tests. Should this be the one from s3 sdk?

Reply:
it's thrown in the implementation of the s3-virtual-schema